### PR TITLE
updated ReadMe

### PR DESCRIPTION
**Pull request by cosminpopescu14**
Added:
* SQL Fundamentals
* Link to Razor template engine to docs
* Quartz.NET

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -151,6 +152,7 @@ If you like or are using this project to learn or start your solution, please gi
     - [HangFire](https://github.com/HangfireIO/Hangfire)
     - [Coravel](https://github.com/jamesmh/coravel)
     - [Fluent Scheduler](https://github.com/fluentscheduler/FluentScheduler)
+    - [Quartz.NET](https://www.quartz-scheduler.net/)
```
Comment (MoienTajik) [inline ReadMe.md]: As other developers experience and myself too, Quartz is not really a good task scheduler and it's too complicated. That's why we don't put that in the roadmap.

Comment (cosminpopescu14) [reply ReadMe.md]: Ok !
I will remove Quartz.NET

Comment (cosminpopescu14) [reply ReadMe.md]: I removed it.

**Comment by MoienTajik** [comment]
> Removed Quartz.NET

Could you update XML and graph image too?

**Comment by cosminpopescu14** [comment]
I updated the xml and diagram

**Pull request merged**